Dockerfile.pachctl

```diff
@@ -10,3 +9,6 @@ ARG LD_FLAGS
 RUN --mount=type=cache,target=/go/pkg/mod \
```

Comment:
nit:  can we start run statements with the command instead of flags? I think it's a more standard usage

Reply:
These are flags for BuildKit. AFAIK it's the only way to specify them; otherwise, docker would think the flags are for the `go install` command.